eng/pipelines/templateUnofficial.yml

```diff
@@ -0,0 +1,29 @@
+
+parameters:
+  - name: templatePath
+    type: string
+    default: no
+  - name: stages
+    type: stageList
+  - name: containers
+    type: object
+
+resources:
+  repositories:
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+
+extends:
+  template: v1/1ES.Unofficial.PipelineTemplate.yml@1ESPipelineTemplates
```

Comment:
If you make this a global variable like it was defined in the original change, I believe we could just move official and unofficial into one.